css/disciplines-slider.styl

```diff
@@ -0,0 +1,123 @@
+DISCIPLINE_COLOR_ALL = #0072FF
```

Comment:
we have the same colors declared in the previous file. Should include that instead to avoid repetition.